distributionviewer/core/static/css/lib.styl

```diff
@@ -17,6 +17,27 @@ $lrg-padding = 2 * $med-padding;
 $chart-preview-height = 250px;
 $chart-preview-width = 300px;
 
+/*
+Population line colors
+
+Colors selected by playing around with [1] with $mild-blue as the base color. To
+be honest, I don't know much about color theory, but these looked like they fit
+pretty well with our colorscheme.
+
+[1] http://www.colorschemer.com/online.html
+
+- openjck
+*/
+$max-populations = 8;
```

Comment:
I have no problem with this approach although d3 does let us pick from its own set of predefined colors. See `d3Scale.category10()` for example. This isn't even a nitpick don't redo your work here but it helped me out to know that on the Marketplace Stats project once.